src/middlewares/wrapper.ts

```diff
@@ -22,8 +26,8 @@ export async function wrapMiddlewaresForSSR<P>(
   handler: Handler = () => Promise.resolve(),
 ): Promise<GetServerSidePropsResult<P>> {
   const result = await wrapMiddlewares(middlewares, handler)(
-    context.req as ExtendedRequest,
-    context.res,
+    context.req as NextApiRequest,
+    context.res as NextApiResponse,
```

Comment:
are those needed?

Reply:
They are needed, because the request and response from the context are not typed with Next.js custom type:
![image](https://user-images.githubusercontent.com/31956107/104030873-566b8f80-51cc-11eb-8753-5011afbba370.png)